README.md

```diff
@@ -57,6 +57,10 @@ specific tagged version of the package would not make any sense.
 
 This package is therefore only suited for installation in the root of your deployable project.
 
+## How to Deal with an "Insecure" Package
+
+If roave/security-advisories conflicts with some package, there's no way you could override this, not even temporarily. So if you cannot update the "insecure" package at the moment, you need to remove roave/security-advisories from your `composer.json`.
```

Comment:
Not the way forward - the idea is to focus on upgrade, not on removing safety checks :P

Reply:
If you can't upgrade at the moment, you can't upgrade at the moment. So the only question is: Is there a way to still get warning about **other** packages, or not.
So instead of discussing this in issues, I think it's better to explain it for everybody.